packages/kbn-shared-ux-components/src/empty_state/assets/index.tsx

```diff
@@ -1,24 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import React from 'react';
-
-import { EuiPanel } from '@elastic/eui';
-
-import { withSuspense } from '@kbn/shared-ux-utility';
-
-export const LazyDataViewIllustration = React.lazy(() =>
-  import('./data_view_illustration').then(({ DataViewIllustration }) => ({
-    default: DataViewIllustration,
-  }))
-);
-
-export const DataViewIllustration = withSuspense(
-  LazyDataViewIllustration,
-  <EuiPanel color="subdued" style={{ width: 226, height: 206 }} />
```

Comment:
I would argue that we shouldn't wait until the illustration is available to load the entire empty prompt. There's a significant lag that loading this illustration adds (on slow networks) that I think the user would benefit more in seeing the content more quickly than waiting just for an image to load.

Reply:
I'll revert, thanks!

Reply:
Loading states are such a balance between getting user the content they need as quickly as possible and not having the page contents jump/re-layout for each individual piece.

Reply:
That's odd, because I definitely saw a difference when working on this fallback in #https://github.com/elastic/kibana/pull/132376. Did something else in this PR make it so that the illustration is no longer async?

Reply:
Nope, discovered after a few experiments on my side... I have a fix that makes it lazy!